src/olympia/migrations/1179-utf8mb4.sql

```diff
@@ -0,0 +1,15 @@
+ALTER TABLE `editor_subscriptions` COMMENT '';
```

Comment:
ugh, is this the syntax to clear a comment?

Reply:
Yeah, I couldn't find another way to do it...